{"identifier":{"url":"doc:\/\/codes.soto.soto-core\/documentation\/SotoCore\/AWSServerError\/context","interfaceLanguage":"swift"},"hierarchy":{"paths":[["doc:\/\/codes.soto.soto-core\/documentation\/SotoCore","doc:\/\/codes.soto.soto-core\/documentation\/SotoCore\/AWSServerError"]]},"kind":"symbol","abstract":[{"text":"additional context information related to the error","type":"text"}],"variants":[{"traits":[{"interfaceLanguage":"swift"}],"paths":["\/documentation\/sotocore\/awsservererror\/context"]}],"primaryContentSections":[{"kind":"declarations","declarations":[{"languages":["swift"],"tokens":[{"text":"let","kind":"keyword"},{"text":" ","kind":"text"},{"kind":"identifier","text":"context"},{"kind":"text","text":": "},{"text":"AWSErrorContext","preciseIdentifier":"s:8SotoCore15AWSErrorContextV","identifier":"doc:\/\/codes.soto.soto-core\/documentation\/SotoCore\/AWSErrorContext","kind":"typeIdentifier"},{"text":"?","kind":"text"}],"platforms":["macOS"]}]}],"schemaVersion":{"minor":3,"patch":0,"major":0},"metadata":{"modules":[{"name":"SotoCore"}],"role":"symbol","symbolKind":"property","roleHeading":"Instance Property","title":"context","fragments":[{"kind":"keyword","text":"let"},{"text":" ","kind":"text"},{"text":"context","kind":"identifier"},{"text":": ","kind":"text"},{"kind":"typeIdentifier","preciseIdentifier":"s:8SotoCore15AWSErrorContextV","text":"AWSErrorContext"},{"kind":"text","text":"?"}],"externalID":"s:8SotoCore14AWSServerErrorV7contextAA15AWSErrorContextVSgvp"},"sections":[],"references":{"doc://codes.soto.soto-core/documentation/SotoCore/AWSErrorContext":{"type":"topic","url":"\/documentation\/sotocore\/awserrorcontext","title":"AWSErrorContext","kind":"symbol","role":"symbol","navigatorTitle":[{"kind":"identifier","text":"AWSErrorContext"}],"fragments":[{"kind":"keyword","text":"struct"},{"kind":"text","text":" "},{"text":"AWSErrorContext","kind":"identifier"}],"identifier":"doc:\/\/codes.soto.soto-core\/documentation\/SotoCore\/AWSErrorContext","abstract":[{"type":"text","text":"Additional information about error"}]},"doc://codes.soto.soto-core/documentation/SotoCore/AWSServerError/context":{"url":"\/documentation\/sotocore\/awsservererror\/context","kind":"symbol","title":"context","abstract":[{"type":"text","text":"additional context information related to the error"}],"fragments":[{"kind":"keyword","text":"let"},{"text":" ","kind":"text"},{"text":"context","kind":"identifier"},{"text":": ","kind":"text"},{"kind":"typeIdentifier","text":"AWSErrorContext","preciseIdentifier":"s:8SotoCore15AWSErrorContextV"},{"text":"?","kind":"text"}],"role":"symbol","type":"topic","identifier":"doc:\/\/codes.soto.soto-core\/documentation\/SotoCore\/AWSServerError\/context"},"doc://codes.soto.soto-core/documentation/SotoCore/AWSServerError":{"title":"AWSServerError","kind":"symbol","abstract":[],"fragments":[{"text":"struct","kind":"keyword"},{"kind":"text","text":" "},{"kind":"identifier","text":"AWSServerError"}],"identifier":"doc:\/\/codes.soto.soto-core\/documentation\/SotoCore\/AWSServerError","type":"topic","role":"symbol","navigatorTitle":[{"text":"AWSServerError","kind":"identifier"}],"url":"\/documentation\/sotocore\/awsservererror"},"doc://codes.soto.soto-core/documentation/SotoCore":{"type":"topic","identifier":"doc:\/\/codes.soto.soto-core\/documentation\/SotoCore","kind":"symbol","url":"\/documentation\/sotocore","abstract":[{"type":"text","text":"The core framework for Soto, a Swift SDK for AWS"}],"role":"collection","title":"SotoCore"}}}